extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking.tsx

```diff
@@ -28,7 +28,7 @@ function PanelMeasurementTableTracking({
   });
 
   useEffect(() => {
-    let updatedMeasurementFilters = { ...measurementFilters };
+    const updatedMeasurementFilters = { ...measurementFilters };
     if (trackedMeasurements.matches('tracking') && trackedStudy) {
       updatedMeasurementFilters.measurementFilter = filterTracked(trackedStudy, trackedSeries);
```

Comment:
Can you fix the measurementFilter to be just a single filter, and make it independent of the tracked study.